src/test/java/Pages/HotelsHomepage.java

```diff
@@ -24,29 +34,70 @@
  public void clickSignInLink(){
      signInLocator.click();
  }
-
-    public void enterEmail (){
-     emailLocator.sendKeys("asdf@mail.com");
+ public void enterEmail (){
+     emailLocator.sendKeys("a^df@mail.com");
```

Comment:
Instead of hard coding email address here, you can make this method parameterized and pass the text from Feature file step.